server_status/views.py

```diff
@@ -152,6 +153,40 @@ def get_celery_info():
     return {"status": UP, "response_microseconds": (datetime.now() - start).microseconds}
 
 
+def get_certificate_info():
+    """
+    checks app certificate expiry status
+    """
+    certificate_status = {
+        'app_cert_expires': '',
+        'status': DOWN
+    }
+    if not getattr(settings, 'MIT_WS_CERTIFICATE_FILE'):
+        return certificate_status
+
+    try:
+        with open(settings.MIT_WS_CERTIFICATE_FILE, "rt") as certificate:
```

Comment:
@pdpinch @mitodl/devops is this right settings?

Reply:
Might be better to use the `MIT_WS_CERTIFICATE` env variable instead?:
```python
MIT_WS_CERTIFICATE=os.environ.get('MIT_WS_CERTIFICATE', None)
```

and then read in the contents via something like this(based on https://github.com/mitodl/odl-video-service/blob/master/odl_video/envs.py#L155):
```python
certificate = settings.MIT_WS_CERTIFICATE.encode().decode('unicode_escape').encode()
```